pkg/operator/resources/cluster/controller.go

```diff
@@ -209,14 +187,14 @@ func GetControllerPermissions() []rbacv1.PolicyRule {
 				"customresourcedefinitions",
 			},
 			Verbs: []string{
-				"*",
+				"get",
```

Comment:
This appears to have broken something in the controller, I see this constantly now:
```
E1120 20:47:45.837940       1 reflector.go:134] pkg/client/informers/externalversions/factory.go:117: Failed to list *v1beta1.CustomResourceDefinition: customresourcedefinitions.apiextensions.k8s.io is forbidden: User "system:serviceaccount:cdi:cdi-sa" cannot list resource "customresourcedefinitions" in API group "apiextensions.k8s.io" at the cluster scope
```